mocks/mocks.go

```diff
@@ -25,6 +27,21 @@ type ErrorReporter interface {
 	Errorf(string, ...interface{})
 }
 
+type ValueChecker func(val []byte) error
+
+func generateRegexpChecker(re string) func([]byte) error {
```

Comment:
not exported, so if you want to actually use it in your app you won't be able to... I think it might just be easier to implement this directly in your app instead of here anyway